src/migrations/20210407170317_statuses_account_index.js

```diff
@@ -0,0 +1,17 @@
+const kIndexAscName = 'idx_tweets_id_asc_account';
+const kIndexDescName = 'idx_tweets_id_desc_account';
+const kTable = 'statuses';
+
+exports.up = async (knex) => {
+  return knex.raw(
+    `CREATE INDEX EXISTS ${kIndexAscName} on ${kTable} using BTREE (id ASC, account);` +
```

Comment:
CREATE INDEX IF NOT EXISTS

Reply:
Fixed

Reply:
@BrRenat @AVVS - I suggest to move this migration [to the existing one](https://github.com/makeomatic/mservice-social/blob/master/src/migrations/20200508111511_statuses_account_column.js). 
If the service is deployed from scratch, this migration will create the `account` column with an indices at once.